webview-ui/src/components/chat/ModelPickerModal.tsx

```diff
@@ -460,157 +592,200 @@ const ModelPickerModal: React.FC<ModelPickerModalProps> = ({ isOpen, onOpenChang
 									</Tooltip>
 								</IconToggles>
 							</SettingsHeader>
+							{/* Thinking budget slider - shown when model supports thinking, greyed out when disabled */}
+							{supportsThinking && (
```

Comment:
should we only show the slider when thinking is enabled and the model supports thinking? Like we do when the plan act mode is enabled that's when we show the plan mode selector?